stats.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional, Dict, Union
+
+StatsAnnot = Dict[str, Union[int, float]]
+
+class Stats:
+    """
+    Champions, items, and rune shards can all be considered to have stats.
+    Stats in league all have a fixed way of being stacked, hence why it makes sense to define a class.
+    """
+
+    def __init__(self, stat_dict: Optional[StatsAnnot] = None):
+        if stat_dict is None:
+            stat_dict = {}
+            
+        for name, value  in stat_dict.items():
+            setattr(self, name, value)
```

Comment:
I would rather store an underlying dictionary, and overight the `__getattr__` method to implement this mechanism.

Reply:
ok I see